### ISSUE TITLE
Runwar tries to open HTTP URL even when HTTP is turned off

### ISSUE THREAD

**Issue by bdw429s** [cfmlprojects/runwar#17]
If I start a CommandBox sever with HTTPS enabled and HTTP disabled, the browser doesn't open and the log shows a timeout trying to open the HTTP URL, which obviously won't work.

**Comment by denuno** [comment]
Hrmm, I'll take a look at fixing this, but the URLs aren't really coming from runwar, they're passed in as arguments or are part of the tray config json file... still, I can see the appeal of not having to use separate configurations for the URLs, so hopefully it won't be too hard of a tweak.

**Comment by bdw429s** [comment]
Ahh, I think I see where it's coming from.  I never knew that was passed to runwar.  Let me see if I can fix this from the CommandBox end.

**Comment by bdw429s** [comment]
OK,  I'm passing the SSL URL to runwar now, but it's still not opening the browser.  Looking at the server log, I think something inside Runwar is still trying to use the HTTP port.

```
Waiting upto 35 seconds for 127.0.0.1:63456...
could not open browser to...https://127.0.0.1:1443... timeout...
```

**Comment by denuno** [comment]
I'd transposed an argument name, so this should be fixed in the next build.

**Comment by denuno** [comment]
I also have it automatically switch the URL to https if SSL is enabled, FWIW.

**Comment by bdw429s** [comment]
Sweet thanks.  I made the same change in CommandBox. If SSL is enabled, I favor that over HTTP.

**Comment by denuno** [comment]
This should be fixed in 3.4.6 too
